src/WpfMath/Atoms/RowAtom.cs

```diff
@@ -81,6 +79,11 @@ internal RowAtom(SourceSpan source, IEnumerable<Atom> elements)
 
         public RowAtom Add(Atom atom)
         {
+            if (atom is null)
+            {
+                return new RowAtom(this.Source, this.PreviousAtom, this.Elements);
```

Comment:
I do not like that I had to write the following

Reply:
I understand that. I'll mark this as a code that will be improved when porting to C# 8: we should really mark this all as non-nullable, but I can see that it solves the problem at the moment.